kclvm/ast/src/ast.rs

```diff
@@ -213,6 +215,17 @@ impl Program {
             None => vec![],
         }
     }
+
+    pub fn pos_to_stmt(&self, pos: &Position) -> Option<Node<Stmt>> {
```

Comment:
Add more documents to explain why it is necessary to obtain AST from pos in reverse and its functions.